lib/main_ltp.pm

```diff
@@ -32,6 +32,14 @@ sub load_kernel_tests {
     loadtest_kernel "../installation/bootloader" if is_pvm;
 
     if (get_var('INSTALL_LTP')) {
+        if (is_alp) {
+            loadtest 'microos/disk_boot';
+            loadtest 'update/zypper_clear_repos';
```

Comment:
for these 3 tests, you can use: `replace_opensuse_repos_tests if is_repo_replacement_required;` (also need to import it from main_common)

Reply:
Not sure, if i understand. Do you mean to add that line after zypper actions?

Reply:
`replace_opensuse_repos_tests` schedules all the modules that handle repo replacement according to test suite variables.

Reply:
Thanks, updated.